File: backend/src/main/java/com/java/NBE4_5_1_7/domain/community/post/dto/PostListResponseDto.java
package com.java.NBE4_5_1_7.domain.community.post.dto;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class PostListResponseDto {
    private Long postId;
    private String title;
    private String author;
    private LocalDateTime createdAt;
}

File: backend/src/main/java/com/java/NBE4_5_3_7/domain/community/post/dto/PostListResponseDto.kt
package com.java.NBE4_5_3_7.domain.community.post.dto

import java.time.LocalDateTime

class PostListResponseDto {
    var postId: Long? = null
    var title: String? = null
    var author: String? = null
    var createdAt: LocalDateTime? = null

    constructor()

    constructor(postId: Long?, title: String?, author: String?, createdAt: LocalDateTime?) {
        this.postId = postId
        this.title = title
        this.author = author
        this.createdAt = createdAt
    }
}